addon/components/array-container.js

```diff
@@ -2,61 +2,57 @@ import 'ember-frost-bunsen/typedefs'
 
 import _ from 'lodash'
 import Ember from 'ember'
+const {A, Component} = Ember
 import computed, {readOnly} from 'ember-computed-decorators'
-
-import layout from './template'
 import PropTypeMixin, {PropTypes} from 'ember-prop-types'
-import {getLabel} from '../utils'
+import {deemberify, getLabel} from '../utils'
```

Comment:
I think `deemberify` should only be necessary for the setting of the initial state. That way we can pass an Ember-Data record to bunsen as our seed value and still have it behave correctly.

Another improvement I think we can make to prevent re-renders is to only pass value down as an initial value OR in the case that we know we want to directly change an input's value since input components maintain their own state already.